utils/land-patterns.stanza

```diff
@@ -1102,8 +1102,8 @@ public defn make-quad-landpattern (num-pins-west: Int,  ; the number of pins on
                                    pitch-south:Double,  ; the pitch on the south face
                                    pitch-east:Double,   ; the pitch on the east face
                                    pitch-north:Double,  ; the pitch on the north face
-                                   component-length: Toleranced, ; the distance from pad-edge to pad-edge on the exterior of the part in the x direction
-                                   component-width: Toleranced, ; the distance from pad-edge to pad-edge on the exterior of the part in the y direction
+                                   lead-span-1: Toleranced, ; the distance from pad-edge to pad-edge on the exterior of the part in the x direction
```

Comment:
why `lead-span-1` instead of `lead-span-x`?

Reply:
Was basing it off the doc shared in slack. We can use `x` instead.